engine.go

```diff
@@ -9,14 +9,28 @@ import (
 	"gopkg.in/src-d/go-mysql-server.v0/sql/parse"
 )
 
+// Config for the Engine.
+type Config struct {
+	// VersionPostfix to display with the `VERSION()` UDF.
+	VersionPostfix string
+}
+
 // Engine is a SQL engine.
 type Engine struct {
 	Catalog  *sql.Catalog
 	Analyzer *analyzer.Analyzer
 }
 
-// New creates a new Engine
-func New(c *sql.Catalog, a *analyzer.Analyzer, versionPostfix string) *Engine {
+// New creates a new Engine with custom configuration. To create an Engine with
+// the default settings use `NewDefault`.
+func New(c *sql.Catalog, a *analyzer.Analyzer, cfg *Config) *Engine {
+	var versionPostfix = "go-mysql-server"
```

Comment:
shall we just do not register the "VERSION()" function if that config property is empty? Or maybe just return the MySQL version and that's it

Reply:
That way, `NewDefault` will always return just the MySQL version